persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_29/Ballerina.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+org = "foo"
+name = "persist_generate_29"
+version = "0.1.0"
+distribution = "2201.3.0"
+
+[build-options]
+observabilityIncluded = false
+
+
```

Comment:
```suggestion
```

Reply:
Updated in [8458b4a](https://github.com/ballerina-platform/persist-tools/pull/84/commits/8458b4a70f4b0fd2252b20bd726f50600b0dda06)